config_src/external/smartredis/MOM_smartredis_MEKE.F90

```diff
@@ -0,0 +1,61 @@
+!> Contains routines that contain dummy routines for the smart
+module MOM_smartredis_meke
+
```

Comment:
Needs the MOM6 license one-liner.  This is the stub which lives in MOM6 so must hae the MOM6 license. The actual code that replaces the stub can use whatever license you like.